### ISSUE TITLE
Use icons from mipmap resources

### ISSUE THREAD

**Issue by polyn0m** [phonegap/phonegap-plugin-push#1314]
### Expected Behaviour

Use icons from mipmap resources. In [cordova-android@6.0.x](https://github.com/apache/cordova-android/tree/6.0.x) application uses mipmap icons (CB-8722), but plugin use only drawable resources.
### Actual Behaviour

Use only drawable resources
#### Steps to Reproduce
- Install [cordova-android@6.0.x](https://github.com/apache/cordova-android/tree/6.0.x)
- Create simple application
- Install plugin
- Replace icons in mipmap folder
- Send push to device
#### Cordova CLI version and cordova platform version

```
cordova --version  - 6.4.0
cordova platform version android  - 6.0.0
```
#### Plugin version

```
1.8.3
```

**Comment by polyn0m** [comment]
@macdonst what means labels `first time contributor` and `good for beginners`?

**Comment by macdonst** [comment]
@polyn0m just trying to identify issues that would be good for other to fix that aren't terribly difficult to work on.

**Comment by keithdmoore** [comment]
@polyn0m Did you happen to figure out a work around?

**Comment by keithdmoore** [comment]
I just downgraded to Cordova 6.3.1 and cordova-android 5.2.2.

**Comment by polyn0m** [comment]
@keithdmoore yep, I'm also downgraded to cordova-android@5.2.2, but use cordova@6.0.x May be I realize this enhancement, but now i don't have time for that.

**Comment by cleitonjar** [comment]
Any news?

Cordova CLI: 6.5.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
ios-deploy version: 1.9.0
ios-sim version: 6.0.0
OS: macOS Sierra
Node Version: v5.10.1
Xcode version: Xcode 8.3.3 Build version 8E3004b

**Comment by macdonst** [comment]
Currently fixed in master, soon to be in release 2.1.0.
